doc/gov/chapters/chapter05.md

```diff
@@ -33,8 +33,19 @@ Being on the voting representative list **doesn't** mean the right to vote. The
 
 <sup>2)</sup>The criteria will be checked and the voting representative list will be updated prior any voting event when the voting process kicks in.
 
-<sup>3)</sup> Individuals who don't present an organisation will not have voting right.
+<sup>3)</sup> Individuals who don't represent an organisation will not have voting right.
```

Comment:
Should be "voting rights"

Reply:
fixed in https://github.com/cntt-n/CNTT/pull/897/commits/f4c5fad87b037ae61d203f946da1c8746f0ae34f